packages/api/src/domain/medical/patient-demographics.ts

```diff
@@ -276,10 +284,8 @@ export function normalizeAddress({
             .replaceAll("avenue", "ave");
         }) ?? [],
     city: city?.trim().toLowerCase() ?? "",
-    state: state?.trim().toLowerCase().slice(0, 2) ?? "",
-    zip: stripNonNumericChars(zip ?? "")
-      .trim()
-      .slice(0, 5),
+    state: normalizeUSStateForAddressSafe(state ?? "")?.toLowerCase() ?? "",
+    zip: normalizeZipCodeNewSafe(zip ?? "") ?? "",
```

Comment:
Assuming these fallbacks to empty string are required? It's a bad way to represent empty value - ideally it would be represented with `undefined`.

Reply:
yes